CONTRIBUTING.md

```diff
@@ -80,6 +80,17 @@ Please use 4-space indentation to trigger code parsing, rather than [fenced code
             std::cout << "This is code.\n";
         }
 
+#### Coding style
+
+Please use [Stroustrup](http://www.stroustrup.com/Programming/PPP-style.pdf) style for naming, like so:
+
+    This is some document text, with an example below:
+
+    class Foo_bar
+    {
+        void do_something(int arg_0);
```

Comment:
I haven't read the Stroustrup doc above, but my guess is that `arg_0` is not really usual. May I suggest just `arg` or omit the argument name?

Reply:
Thanks for commenting.